ansible/roles/build-grsec-kernel/tasks/fetch_files.yml

```diff
@@ -1,50 +1,24 @@
 ---
-- name: ensure download dir exists
+- name: Create directory for downloaded files.
   file:
     state: directory
-    dest: "{{download_directory}}"
+    dest: "{{ download_directory  }}"
 
-- name: fetch linux kernel and grsecurity patches
+- name: Fetch Linux kernel tarball and signature.
   get_url:
-      url: "{{item.url}}"
-      dest: "{{download_directory}}/{{item.filename}}"
-      sha256sum: "{{item.sha256sum}}"
+      url: "{{ item.url  }}"
+      dest: "{{ download_directory }}/{{ item.filename }}"
+      sha256sum: "{{ item.sha256sum }}"
   with_items:
-    - "{{linux_tarball}}"
-    - "{{linux_tarball_signature}}"
-    - "{{grsecurity_patch}}"
-    - "{{grsecurity_patch_signature}}"
+    - "{{ linux_tarball }}"
+    - "{{ linux_tarball_signature }}"
 
-- name: clone ubuntu overlay git repository (slow!)
-  git:
-    repo: git://kernel.ubuntu.com/ubuntu/ubuntu-trusty.git
-    dest: "{{download_directory}}/ubuntu-trusty"
-    force: yes
-    # since this clone takes a long time, just grab
-    # the first layer. can then pull manually to update.
-    depth: 1
-    # definitely don't want to do this, but only
-    # other option is scanning kernel.ubuntu.com
-    # and accepting whatever is returned
-    accept_hostkey: yes
-  tags:
-    - git
-  when: target_os == 'ubuntu'
+- name: Fetch grsecurity patch and signature.
+  debug: msg=hello
 
-- name: copy kernel package files
-  shell: cp -a /usr/share/kernel-package {{download_directory}}/ubuntu-package
-  args:
-    creates: "{{download_directory}}/ubuntu-package"
-  when: target_os == 'ubuntu'
 
-- name: copy ubuntu overlay control files
-  shell: >
-     cp {{download_directory}}/ubuntu-trusty/debian/control-scripts/p* \
-        {{download_directory}}/ubuntu-package/pkg/image/
-  when: target_os == 'ubuntu'
+- name: Fetch grsecurity patch and signature.
```

Comment:
Placeholder during development! The actual logic has moved to `tasks/fetch_grsecurity_files.yml`. Cleaned up the task lists in 989b7be. Thanks for catching!